runWPT.sh

```diff
@@ -1,31 +1,60 @@
-#!/bin/sh
+#!/bin/bash
+# shellcheck disable=SC2155
 
-LOG_DIR="logs"
-LOG_FILE="$LOG_DIR/$(date +%s).log"
-readonly LOG_DIR LOG_FILE
+log() {
+  echo "($0) $*"
+}
+
+usage() {
+  log "Usage: [HEADLESS=<true | false>] [UPDATE_EXPECTATIONS=<true | false>] $0 [webdriver/tests/bidi/[...]]"
+}
+
+if [[ "$1" == "-h" || "$1" == "--help" ]]; then
+  usage
+  exit 0
+fi
+
+readonly HEADLESS="${HEADLESS:-true}"
+readonly MANIFEST="${MANIFEST:-MANIFEST.json}"
+readonly PRODUCT="${PRODUCT:-chromium}"
+readonly UPDATE_EXPECTATIONS="${UPDATE_EXPECTATIONS:-true}"
+readonly WPT_BROWSER_PATH="${WPT_BROWSER_PATH:-/Applications/Google Chrome Dev.app/Contents/MacOS/Google Chrome Dev}"
+readonly WPT_REPORT="${WPT_REPORT:-wptreport.json}"
+
+if [[ "$HEADLESS" == "true" ]]; then
+  readonly WPT_METADATA="wpt-metadata/mapper/headless"
+else
+  readonly WPT_METADATA="wpt-metadata/mapper/headful"
+fi
+
+log "Running WPT in headless=$HEADLESS mode..."
 
 # Go to the project root folder.
 (cd "$(dirname "$0")/" && \
-# Create a folder to store logs.
-mkdir -p "$LOG_DIR" && \
-npm run build && \
 ./wpt/wpt run \
   --binary "$WPT_BROWSER_PATH" \
   --webdriver-binary runBiDiServer.sh \
-  --webdriver-arg="--verbose" \
```

Comment:
These were invalid! They are not correct options for Bidi Server (only for classic chromedriver).